tools/genkconfig.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
 import argparse
 import re
-import jinja2
 from glob import glob
 from pathlib import Path
 
+import jinja2
```

Comment:
Nit: Do we need this change?

Reply:
Oh my bad. I forgot to save the file without formatting.